src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -27,11 +28,21 @@ interface BenchmarkFormEntry {
   [key: string]: string | number | "";
 }
 
+const BenchmarkStepperIcon = (stepIconProps: StepIconProps) => {
```

Comment:
This new component takes the same properties as `StepIcon`, so it can be safely passed as the `StepIconComponent` of a `StepLabel` without generating warnings.